synchronization/src/movebis/java/de/cyface/synchronization/CyfaceSyncAdapter.java

```diff
@@ -26,10 +26,11 @@
 import android.util.Log;
```

Comment:
I've an open task to remove / merge all duplicate code from both flavours to the general folder.